src/System.Management.Automation/engine/runtime/MutableTuple.cs

```diff
@@ -364,7 +364,7 @@ internal static IEnumerable<PropertyInfo> GetAccessProperties(Type tupleType, in
         {
             // ContractUtils.RequiresNotNull(tupleType, "tupleType");
 
-            if (index < 0 || index >= size) throw new ArgumentException("index");
+            if (index < 0 || index >= size) throw new ArgumentException(null, nameof(index));
```

Comment:
The same.

Reply:
`index` is value type `int` so may not be `null`.

Reply:
Same as above.